.changeset/brown-forks-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fixed a problem where the setting `Livechat_show_agent_email` was not being used by the back when returning agent's info
```

Comment:
Do you think, since this is supposed to be for users, we should instead of using setting ids here, provide setting paths like `Omnichannel > Email > Show Agent Email` ?

Reply:
:hmmm: not really sure. But I do agree we can remove the _ from the setting name lol